# ONT Methylation Benchmarking Datasets

## Description
ONT Methylation Benchmarking Datasets are generated to benchmark existing methylation-calling tools on the Oxford Nanopore sequencing platform using their recent R10.4.1 flowcell chemistry. It spans a diverse range of species, including bacteria (E. coli, H. pylori J99, H. pylori 26695, A. variabilis, T. denticola), plants (Rice, Arabidopsis), and mammals (mouse, human).
In addition, the dataset includes EMSeq data for E. coli, plant, and mouse samples, which can serve as ground truth for methylation studies. It also provides unmethylated whole-genome amplified (WGA) DNA for H. pylori 26695 and a dam- dcm- double mutant (DM) of E. coli that lacks canonical 5mC and 6mA methylation. These variants, together with their wild-type counterparts, offer value for both training and benchmarking DNA methylation calling models.

## Update Frequency
Datasets will be updated periodically as additional data is generated.

## License
CC BY-NC 4.0

## Documentation
[Github repo link](https://github.com/SowpatiLab/ont_benchmarking_roda_dataset)

## Managed by
CSIR-Centre for Cellular and Molecular Biology (https://www.ccmb.res.in/)

## Conrtact
[Github repo link](https://github.com/SowpatiLab/ont_benchmarking_roda_dataset)

## How to Cite
Please cite Kulkarni et al. Comprehensive benchmarking of tools for nanopore-based detection of DNA methylation. bioRxiv (2024). doi: https://doi.org/10.1101/2024.11.09.622763 when referencing the ONT methylation benchmarking datasets in publications.

## Usage Examples

### Tutorial
[Github repo link](https://github.com/SowpatiLab/ont_benchmarking_roda_dataset)

## Publications
Comprehensive benchmarking of tools for nanopore-based detection of DNA methylation by Kulkarni et al.